dataset/src/main/scala/frameless/ops/AggregateTypes.scala

```diff
@@ -0,0 +1,19 @@
+package frameless
+package ops
+
+import shapeless._
+
+trait AggregateTypes[V, U <: HList] {
```

Comment:
But it doesn't work unless you use dependent types for `underlying.TypedAggregateT`